test/callback.tests.js

```diff
@@ -583,6 +583,93 @@ describe('callback response_mode: form_post', () => {
     assert.equal(newTokens.refreshToken, '__test_refresh_token__');
   });
 
+  it('should refresh an access token and pass tokenEndpointParams and refresh argument params to the request', async () => {
+    const idToken = makeIdToken({
+      c_hash: '77QmUPtjPfzWtF2AnpK9RQ',
+    });
+
+    const authOpts = {
+      ...defaultConfig,
+      clientSecret: '__test_client_secret__',
+      authorizationParams: {
+        response_type: 'code id_token',
+        audience: 'https://api.example.com/',
+        scope: 'openid profile email read:reports offline_access',
+        tokenEndpointParams: {
+          longeLiveToken: true
+        }
+      },
+    };
+    const router = auth(authOpts);
+    router.get('/refresh', async (req, res) => {
+      const accessToken = await req.oidc.accessToken.refresh({ force: true });
+      res.json({
+        accessToken,
+        refreshToken: req.oidc.refreshToken,
+      });
+    });
+
+    const { tokens, jar } = await setup({
```

Comment:
You can also get the `tokenReqBody` for the code exchange, then assert the `tokenEndpointParams` have been added there too.

```js
const { tokens, jar, tokenReqBody } = await setup({ ... });

assert.match(tokenReqBody, /longeLiveToken=true/);
```